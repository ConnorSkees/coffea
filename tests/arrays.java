public class arrays {

    public static void main(String[] args) {
        int a = 1;
        int[] b = new int[a + 5];
        String[] c = new String[a + a];
        String[] c2 = { "hi", "bye" };

        int[] d = { 1, 2, 3 };
        d[2] = 4;
        int e = d[0];
        int[][] f = { { 1, 2 }, { 3, 4 }, { 5, 6 } };
        int[] g = f[1];
        int h = f[0][1];
        int i = f[f[0][0]][d[h + 1]];
    }
}