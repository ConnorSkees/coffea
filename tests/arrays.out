public class test {
  public static void main(String[] arg0) {
    int i1 = 1;
    int[] arr2 = new int[(i1 + 5)];
    String[] arr3 = new String[(i1 + i1)];
    String[] arr4 = { "hi", "bye" };
    int[] arr5 = { 1, 2, 3 };
    arr5[2] = 4;
    int i6 = arr5[0];
    int[][] arr7 = { { 1, 2 }, { 3, 4 }, { 5, 6 } };
    int[] arr8 = arr7[1];
    int i9 = arr7[0][1];
    int i10 = arr7[arr7[0][0]][arr5[(i9 + 1)]];
    return;
  }
}